repository/i.json

```diff
@@ -236,6 +236,17 @@
 				}
 			]
 		},
+		{
+			"name": "image-link-insertion",
```

Comment:
You might want to consider calling it simply "Image Link Insertion"?

Reply:
Done, thanks for the suggestion.

Reply:
Ah, that looks better. Don't forget that this name will now be the path and identifier for your package, so that affects where Sublime will look for settings files etc.

Reply:
Oh, and you might want to update your command palette entries as well. Let me know when that's done, and we should be ready to ship.